src/main/java/uk/gov/hmcts/reform/wataskmonitor/config/JacksonConfiguration.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.reform.wataskmonitor.config;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.fasterxml.jackson.module.paramnames.ParameterNamesModule;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import org.springframework.http.converter.json.Jackson2ObjectMapperBuilder;
+
+import static com.fasterxml.jackson.databind.DeserializationFeature.READ_UNKNOWN_ENUM_VALUES_USING_DEFAULT_VALUE;
+
+@Configuration
+public class JacksonConfiguration {
```

Comment:
What's the need for this class?

Reply:
I'd rather use the: @JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class) at the class level. It gives you visibility about it immediately.

Reply:
This is the jackson configuration to make sure this service only accepts snake case and reads enums with default values